libs/aws/tests/integration_tests/chat_models/test_bedrock.py

```diff
@@ -309,7 +311,7 @@ def test_chat_bedrock_token_callbacks() -> None:
     """
     callback_handler = FakeCallbackHandlerWithTokenCounts()
     chat = ChatBedrock(  # type: ignore[call-arg]
-        model_id="anthropic.claude-v2", streaming=False, verbose=True
```

Comment:
This test cases is for streaming (as per description). The `streaming=False` was toggled on this [PR](https://github.com/langchain-ai/langchain-aws/pull/119/files#diff-6d14d4a53f73d46e8fd45ed545e2a040c430b34bb79bc02c0843e0e33d3f2c6bR312), toggling back on.